packages/storybook/stories/components/button/documentation.mdx

```diff
@@ -42,39 +42,50 @@ _A **Button** component aims to initiate an action. Its <StorybookLink kind={ OD
 **Buttons** are mainly used for actions, like adding, removing, validating, etc.
 They are used to allow users to interact with the page or its content.
 
-**Buttons** work with other elements on a screen to **surface the most important actions the user wants to take in that context**.
-**They must respect the proximity law** in order to guide the user on the action to be performed and the expected result.
+**Buttons** work with other elements on a screen to surface the most important actions the user wants to take in that context.
+They must respect the proximity law in order to guide the user on the action to be performed and the expected result.
 A **Button** label indicates what happens when the user taps the **Button**, even if it's just to acknowledge something.
 
 **Buttons** don't usually redirect to an external page. For that, see <StorybookLink kind={ ODS_COMPONENTS_TITLE.link } story={ STORY.documentation }>Link</StorybookLink>.
 
+<Heading label="When to use a Link or a Button?" level={ 3 } />
+
+<StorybookLink kind={ ODS_COMPONENTS_TITLE.link } story={ STORY.documentation }>Links</StorybookLink> and **Buttons** serve different purposes, and using them correctly improves both accessibility and user experience:
+
+* Use a Link when the action navigates the user to another page or resource, either within your application or to an external site
+* Use a **Button** when the action performs a function or triggers a behavior on the current page, like submitting a form, opening a modal, or toggling a menu
+
+A **Button** should not mimic a Link. It can lead to confusion for users and assistive technologies, as **Buttons** are not typically expected to handle navigation.
```

Comment:
link
(same for all in page where it's not an actual mention of the component but a general link)